documentation/NUnit1001.md

```diff
@@ -0,0 +1,51 @@
+# NUnit1001
+## Find Incorrect TestCaseAttribute Usage
+
+| Topic    | Value
+| :--      | :--
+| Id       | NUnit1001
+| Severity | Error
+| Enabled  | True
+| Category | Structure
+| Code     | [TestCaseUsageAnalyzer](https://github.com/nunit/nunit.analyzers/blob/master/src/nunit.analyzers/TestCaseUsage/TestCaseUsageAnalyzer.cs)
+
+
+## Description
```

Comment:
No description provided in the descriptors. I included this header with empty value in the generated stubs.

Reply:
I think it is fine. We should have descriptions in this project